knowledge-base/fileselect-disable-onselect.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to change the FileSelect button's caption and disable it during selection
+description: Hot to change FileSelect button's caption, while selecting and uploading. How to disable it during selection.
+type: how-to
+page_title: How to disable the FileSelect, while selecting and changing its caption's button.
```

Comment:
Double-check your page_title as it seems to convey a different meaning than what was already described in the title and the description.